src/components/DriverVersionsCatalog.module.css

```
@@ -322,7 +322,7 @@
   line-height: 1.2;
   font-weight: 800;
   color: var(--ifm-color-emphasis-900);
-  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)